gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-webhook/pom.xml

```diff
@@ -51,6 +51,12 @@
             <artifactId>gravitee-gateway-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>io.gravitee.apim.gateway.handlers</groupId>
+            <artifactId>gravitee-apim-gateway-handlers-api</artifactId>
```

Comment:
I am a bit surprised that a plugin need such a core dependencies to work. What is required for their ?

Reply:
It's required for using VertxHttpClient from gravitee-apim-gateway-http and SslOptions from gravitee-apim-definition-model.
I took example on gravitee-apim-plugin-endpoint-http-proxy dependencies.